internal_policies/suggestion-tags.md

```diff
@@ -0,0 +1,29 @@
+---
+description: List of tags used in the Discord suggestions channel.
+---
+
+# Discord Suggestions Tags
+
+Suggestions channel in Polyfrost discord.
+
+{% embed url="https://discord.com/channels/822066990423605249/1265704794778243276" %}
+
+## All Tags
+
+- `OneConfig UX` - topics related to user accessibility or OneConfig design
+- `OneConfig API` - developer-related API suggestions for OneConfig
+- `Mods` - topics related to Polyfrost Mods.
+- `OneLauncher` - topics related to OneLauncher
+- `Meta` - topics related the Discord or Website
+
+The poster will have the opportunity to choose a tag from the [All Tag](#all-tags) category.
+If they choose the wrong categories, staff will fix it for them.
+
+The Polyfrost devs will then read the post and give feedback.
+If they haven't added a feature state tag, planned, not planned, or implemented, then go add the relevant one.
+
+## Moderator / Support Only Tags
+
+- `Planned / Not Planned` - Whether the suggestion is accepted or not
+- `Implemented` - Whether the feature has been made
+- `Duplicate` - Whether the feature has already been suggested before (regardless of whether it was accepted or not)
```

Comment:
Forgot to put here, but there's a new invalid tag

- `Invalid` - Whether the suggestion is invalid for any reason (e.g. a joke suggestion)

Reply:
Fixed